### ISSUE TITLE
Correct the content of index.md file of machine learning directory.

### ISSUE THREAD

**Issue by srilekha279** [animator/learn-python#1228]
I want make the following changes in this file:  [https://github.com/animator/learn-python/blob/main/contrib/machine-learning/index.md](https://github.com/animator/learn-python/blob/main/contrib/machine-learning/index.md)
1) Change TensorFlow.md to TensorFlow.
2) Change PyTorch.md to PyTorch.

Please assign me this issue.

**Comment by srilekha279** [comment]
Kindly assign the label level1 to this issue.

**Comment by srilekha279** [comment]
Please assign level for my issue.
